lib/rules/declaration-block-no-duplicate-properties/__tests__/index.js

```diff
@@ -18,6 +18,10 @@ testRule(rule, {
       code: "a { color: pink; { &:hover { color: orange; } } }",
       description: "spec nested"
     },
+    {
+      code: "color: pink; @media { color: orange; }",
```

Comment:
As per [the developer guide](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#write-tests):

> Ensure you use standard CSS syntax by default

I don't believe a declaration outside of rule-set is valid. What are we trying to test here?

If it's the content of a `style=""` attribute then how does this differ from the `syntax: html` tests below which are more explicit?